library/src/webapp/js/portal/portal.sidebar.sites.js

```diff
@@ -37,6 +38,10 @@ class SitesSidebar {
 
     const update = (siteId, expanded) => {
 
+      if (! this._isLoggedIn) {
```

Comment:
You could just use portal.user.id directly here. Just check the falsiness of it.

if (!portal.user.id) return;

Reply:
@adrianfish - Thanks. Yes, that would be simpler. I'll force push an update momentarily.